src/main/app/src/app/zaken/zaak-create/zaak-create.component.ts

```diff
@@ -137,9 +133,7 @@ export class ZaakCreateComponent implements OnInit {
                                                        .build();
         this.locatieField.formControl.disable({onlySelf: true});
         const zaaktypeEnInitiator: AbstractFormField[] = [zaaktype];
-        if (this.rechten.toevoegenInitiatorPersoon || this.rechten.toevoegenInitiatorBedrijf) {
-            zaaktypeEnInitiator.push(this.initiatorField);
-        }
+        zaaktypeEnInitiator.push(this.initiatorField);
 
         this.createZaakFields = [
             [titel],
```

Comment:
zaaktypeEnInitiator vervangen door [zaaktype, this.initiatorField] en zaaktypeEnInitiator verwijderen